### PR TITLE
Fixes refunding wizard contracts not giving you back the right amount of spell points

### PR DESCRIPTION
**Pull request by JustSumBody**
Supposed to be five points now, it only gave back one

### CODE REVIEW

**Comment by clusterfack** [comment]
Ok so, no this doesn't make any sense.

It should be giving you back the spell cost, not just 5 points for each spell. Do you not see how that is going to obviously cause a similar bug in the future?

File: code/game/gamemodes/wizard/spellbook.dm [hunk 1]
```diff
@@ -73,7 +73,7 @@
 			to_chat(user, "The contract has been used, you can't get your points back now.")
 		else
 			to_chat(user, "You feed the contract back into the spellbook, refunding your points.")
-			src.uses++
+			src.uses += 5
```
Comment (PJB3005) [inline code/game/gamemodes/wizard/spellbook.dm]: Magic number!

Comment (unid15) [reply code/game/gamemodes/wizard/spellbook.dm]: (it should be Sp_BASE_PRICE instead)

Comment (clusterfack) [inline code/game/gamemodes/wizard/spellbook.dm]: But no it really should be using the spells inherent prices

Comment (unid15) [reply code/game/gamemodes/wizard/spellbook.dm]: artifacts don't keep track of the price. Sp_BASE_PRICE should do as a band aid

Comment (PJB3005) [reply code/game/gamemodes/wizard/spellbook.dm]: > Band aid.

m8

**Review by PJB3005 — APPROVED** [review]

**Comment by JustSumBody** [comment]
Hold up I'm not sure how to actually fix this proper now with what cluster suggested
Would I just make a define specific to it and assign its cost to the define, then use the define in the refund as well?

**Comment by PJB3005** [comment]
@JustSumBody sounds like a plan.

**Comment by unid15** [comment]
That's one way to do it

another, cleaner and more reliable way would be give artifact datums a "refundable" variable and put some code in the spellbook's attackby() that checks if the item can be spawned by an artifact datum, and if it can - take the item away and add some points

**Comment by clusterfack** [comment]
Either of those ways would be fine

**Review by unid15 — APPROVED** [review]

**Pull request merged**